superx/information_extractors/item_info_extractor.py

```diff
@@ -132,7 +150,7 @@ def extract_xml_from_zip_and_parse(self, zip_links):
                 tree = ET.fromstring(xml_file)
                 branch_id = tree.find(store_id).text.lstrip('0')
                 # gets child containing item information
-                info_child_node = tree.getchildren()[-1]  #pylint: disable=deprecated-method
+                info_child_node = tree.getchildren()[-1]  # pylint: disable=deprecated-method
```

Comment:
if pylint says this function is deprecated, why not look for the newer, proper method of doing this?

Reply:
with regards to this, there is a different way of doing, it which is giving it a specific route to the child node. that is the better way of doing it.
the reason I chose to stay with this depreciated method is that it generalizes the problem. the last child will always be the child node that I am looking for in every supermarket XML. 
this was a calculated decision, not a lazy decision.

Reply:
I'll approve it this time, but keep in mind in the future that even if a deprecated function is better for you, it will disappear at some point in any case, which leaves you with code that will stop working.